src/common/Surface.js

```diff
@@ -130,13 +138,42 @@
             .display(this.showTitle())
             .render()
         ;
+
+        var surfaceButtons = element.selectAll(".surface-button").data(this.buttonAnnotations());
+        surfaceButtons.enter().append("g").classed("surface-button",true)
+            .each(function (button, idx) {
+                context._surfaceButtons[idx] = new Icon()
+                .faChar(button["char"])
```

Comment:
I am doing it just for readability (most IDEs ... atleast the ones I am using) will color in the param name a diff color than the surrounding text. If you want me to change this let me know.